### PR TITLE
AT 04.01 | Create New Folder TC

### PR DESCRIPTION
**Pull request by antonbloha**

### CODE REVIEW

**Comment by antonbloha** [comment]
https://trello.com/c/JtI34i2P/270-at-0401-item-folder-create-new-folder

**Review by NataliaKubiak — APPROVED** [review]

File: src/test/java/school/redrover/CreateNewFolder.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        clickOkButton.click();
+
+        WebElement clickSaveButton = getDriver().findElement(By.xpath("//*[@name='Submit']"));
+        clickSaveButton.click();
+
+        WebElement clickDashboardButton = getDriver().findElement(By.xpath("//div[@id='breadcrumbBar']/ol/li[1]/a"));
+        clickDashboardButton.click();
+
+        WebElement checkFolderName = getDriver().findElement(By.xpath("//table[@id='projectstatus']/tbody/tr/td[3]/a/span"));
+        checkFolderName.getText();
```
Comment (judmi) [inline src/test/java/school/redrover/CreateNewFolder.java:35]: What for do you use getText() here? There's no need in it. Just use it in assert

File: src/test/java/school/redrover/CreateNewFolder.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        WebElement newItemLink = getDriver().findElement(By.xpath("//div[@id='tasks']/div[1]/span/a"));
+        newItemLink.click();
+
+        WebElement enterItemName = getDriver().findElement(By.xpath("//input[@name='name']"));
+        enterItemName.sendKeys(name);
+
+        WebElement createFolderButton = getDriver().findElement(By.xpath("//div[@id='j-add-item-type-nested-projects']/ul/li[1]/label/span"));
+        createFolderButton.click();
+
+        WebElement clickOkButton = getDriver().findElement(By.xpath("//*[@id='ok-button']"));
```
Comment (judmi) [inline src/test/java/school/redrover/CreateNewFolder.java:25]: It's better to remove "click" from variable names as in variable you just store a button element. And in the next line code you click it. clickSomeButton is usually a name for some method that clicks on a button, not for a variable that just stores an element

File: src/test/java/school/redrover/CreateNewFolder.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package school.redrover;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+import static org.testng.Assert.assertEquals;
```
Comment (judmi) [inline src/test/java/school/redrover/CreateNewFolder.java:9]: There should be an empty line between imports and class initialization

File: src/test/java/school/redrover/CreateNewFolder.java (hunk 2 above)
Comment (NataliaRamanenka) [reply src/test/java/school/redrover/CreateNewFolder.java:25]: I agree with judmi, It's better to rename WebElements

**Review by ReginaKurtLV — APPROVED** [review]

**Pull request merged**